### ISSUE TITLE
explorer gets stuck at block 840356

### ISSUE THREAD

**Issue by dannybabbev** [ordinals/ord#3651]
Running explorer v0.18.1 in mainnet gets stuck at block 840356 with the following error:

```
Listening on http://0.0.0.0:8080
thread '<unnamed>' panicked at src/index/lot.rs:78:9:
attempt to divide by zero
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by GiantDole** [comment]
okay, so I'm not the only one. seems like unisat and OKX have these issues as well since balance is not updating after confirmed transactions.

https://github.com/ordinals/ord/issues/3652

**Comment by ShadowyCryptoTrader** [comment]
same error here with ord v0.18.1

**Comment by raphjaph** [comment]
Fixed in 0.18.2 and 0.18.3
